code/modules/reagents/reagent_containers/syringes.dm

```diff
@@ -14,6 +14,7 @@
 	possible_transfer_amounts = null //list(5,10,15)
 	volume = 15
 	starting_materials = list(MAT_GLASS = 1000)
+	origin_tech = Tc_BIOTECH  + "=2;" + Tc_COMBAT  + "=1"
```

Comment:
Unforeseen consequences...

Reply:
I looked at all the syringe children just in case. It's fiiiine. Free anti-toxin oh no!